### PR TITLE
feat(testing): OP-stack deployment integration testing.

### PR DESCRIPTION
**Pull request by itsdevbear**

### CODE REVIEW

File: eth/core/state/statedb.go [hunk 1]
```diff
@@ -230,10 +230,6 @@ func (sdb *stateDB) GetStorageProof(_ common.Address, _ common.Hash) ([][]byte,
 	return nil, nil
 }
```
Comment (calbera) [inline eth/core/state/statedb.go:231]: `GetStorageProof` also needs to be deleted so it calls the Plugin

File: testing/op-l2.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+SEQ_KEY="73bead3e3e588be3ae77598160d6a5017acdb008973f21166571e400a4b51246"
+L1_RPC="http://localhost:8545"
+RPC_KIND="any"
```
Comment (Ferret-san) [inline testing/op-l2.sh:5]: `RPC_KIND` should be `basic` to avoid calling non existing methods into the Polaris JSON RPC API

Comment (itsdevbear) [reply testing/op-l2.sh]: fixed

File: eth/core/state/statedb.go (hunk 1 above)
Comment (itsdevbear) [reply eth/core/state/statedb.go:219]: reeeeee its so cooked LMFAO we need MPT

**Pull request closed without merging**